CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.2 - 2020-12-15]
```

Comment:
For SemVer, any changes which aren't bug fixes maybe should bump the minor, so 1.2.0?

Reply:
@adamdmharvey done, thanks